README.md

```diff
@@ -1,33 +1,70 @@
+# Getting Started with Create React App
```

Comment:
[Boulder] Make sure to update this to reflect the status of the project. Keep the original version and improve on it.